H/csGblMtx.h

```diff
@@ -23,93 +23,106 @@
 #ifndef CSOUND_CSGBLMTX_H
 #define CSOUND_CSGBLMTX_H
 
-
 #ifdef HAVE_PTHREAD
 #include <pthread.h>
 
 #ifdef __cplusplus
-extern "C" {
+extern "C"
+{
 #endif
 
-static pthread_mutex_t csound_global_lock_ = PTHREAD_MUTEX_INITIALIZER;
-
-void csoundLock() {
-  pthread_mutex_lock(&csound_global_lock_);
-}
+  static pthread_mutex_t csound_global_lock_ = PTHREAD_MUTEX_INITIALIZER;
 
-void csoundUnLock() {
-  pthread_mutex_unlock(&csound_global_lock_);
-}
+  void
+  csoundLock ()
+  {
+    pthread_mutex_lock (&csound_global_lock_);
+  }
 
+  void
+  csoundUnLock ()
+  {
+    pthread_mutex_unlock (&csound_global_lock_);
+  }
 
 #ifdef __cplusplus
 }
 #endif
 
-#elif defined(_WIN32) || defined (__WIN32__)
+#elif defined(_WIN32) || defined(__WIN32__)
 #define _WIN32_WINNT 0x0600
 #include <windows.h>
 
 #ifdef __cplusplus
-extern "C" {
-  #endif
+extern "C"
+{
+#endif
 
-static INIT_ONCE g_InitOnce = INIT_ONCE_STATIC_INIT;
-static CRITICAL_SECTION* csound_global_lock;
+  static INIT_ONCE g_InitOnce = INIT_ONCE_STATIC_INIT;
+  static CRITICAL_SECTION *csound_global_lock;
 
-static BOOL CALLBACK InitHandleFunction ( PINIT_ONCE InitOnce, PVOID Parameter,
-    PVOID *lpContext) {
+  static BOOL CALLBACK
+  InitHandleFunction (PINIT_ONCE InitOnce, PVOID Parameter, PVOID *lpContext)
+  {
 
-    CRITICAL_SECTION* cs = (CRITICAL_SECTION*) malloc(sizeof(CRITICAL_SECTION));
-    InitializeCriticalSection(cs);
+    CRITICAL_SECTION *cs
+        = (CRITICAL_SECTION *)malloc (sizeof (CRITICAL_SECTION));
+    InitializeCriticalSection (cs);
     *lpContext = cs;
     return 1;
-}
-
+  }
 
-
-void csoundLock() {
+  void
+  csoundLock ()
+  {
     BOOL status;
-    CRITICAL_SECTION* cs;
+    CRITICAL_SECTION *cs;
 
-    status = InitOnceExecuteOnce(&g_InitOnce, InitHandleFunction, NULL, &cs);
-    if (status) {
-      EnterCriticalSection(cs);
-    }
-}
+    status = InitOnceExecuteOnce (&g_InitOnce, InitHandleFunction, NULL, &cs);
+    if (status)
+      {
+        EnterCriticalSection (cs);
+      }
+  }
 
-void csoundUnLock() {
+  void
+  csoundUnLock ()
+  {
 
     BOOL status;
-    CRITICAL_SECTION* cs;
-
-    status = InitOnceExecuteOnce(&g_InitOnce, InitHandleFunction, NULL, &cs);
-    if (status) {
-      LeaveCriticalSection(cs);
-    }
-}
+    CRITICAL_SECTION *cs;
 
+    status = InitOnceExecuteOnce (&g_InitOnce, InitHandleFunction, NULL, &cs);
+    if (status)
+      {
+        LeaveCriticalSection (cs);
+      }
+  }
 
 #ifdef __cplusplus
 }
 #endif
 
 #else /* END WIN32 */
 #ifdef __cplusplus
-extern "C" {
+extern "C"
+{
 #endif
 
-void csoundLock() {
-}
+  void
+  csoundLock ()
```

Comment:
Formatting seems to have taken a step backwards for this PR. Both function definitions and structs got converted to having their opening brace on their own lines, which seems a change from our current conventions.

Reply:
do we have a script? Otherwise it is hard to fix? I used clang-format with -style=GNU. 

If there is a set format that can be applied, it's an easy fix. Otherwise we're stuck.

Reply:
@jpffitch has had a script to format but I don't think it's in the repo? If we go with clang-format it has options we could configure and standardize upon. As it is now, the code formatting differences in recent PRs has definitely increased the difficulty of code reviews. I like the idea of us getting this standardized. 

John, could you point to what settings you have used for code formatting?  I think we've been using 2-space indent, no tabs, then rest I'm unsure there's been a  mix and I think my own preferences (such as always using braces with conditionals) isn't the norm.

Reply:
For now, I reverted the clang-format commit and just corrected the indent of those four files.

Reply:
Can we merge this now if it's only an issue of formatting?